telegram/bot.py

```diff
@@ -1454,7 +1469,8 @@ def send_chat_action(self, chat_id, action, timeout=None, **kwargs):
         """
         Use this method when you need to tell the user that something is happening on the bot's
         side. The status is set for 5 seconds or less (when a message arrives from your bot,
-        Telegram clients clear its typing status).
+        Telegram clients clear its typing status). We only recommend using this method when a
```

Comment:
"Telegram only recommends …". We = PTB

Reply:
I've seen 'We' being used in other places too where it should have been 'Telegram'. Changing it in those places too..